spec/features/viewing_peeps_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'pg'
+require './lib/chitter'
+
+feature 'Viewing peeps' do
+  scenario 'A user can see peeps' do
+    # connection = PG.connect(dbname: 'chitter_test')
+
+    # # Add the test data
+    # connection.exec("INSERT INTO peeps VALUES(1, 'Peep one');")
+    # connection.exec("INSERT INTO peeps VALUES(2, 'Peep two');")
+    # connection.exec("INSERT INTO peeps VALUES(3, 'Peep three');")
+
+    # Bookmark.create(website: "http://www.makersacademy.com")
+    # Bookmark.create(website: "http://www.destroyallsoftware.com")
+    # Bookmark.create(website: "http://www.google.com")
```

Comment:
I didn't think to add the bookmark stuff in comments for easier comparison! smart